node/sc/bridgepool/sorted_map_list.go

```diff
@@ -96,15 +103,21 @@ func (m *ItemSortedMap) Exist(nonce uint64) bool {
 
 // Put inserts a new item into the map, also updating the map's nonce
 // index. If a item already exists with the same nonce, it's overwritten.
-func (m *ItemSortedMap) Put(event itemWithNonce) {
+func (m *ItemSortedMap) Put(event itemWithNonce) error {
 	m.mu.Lock()
 	defer m.mu.Unlock()
 
+	if m.sizeLimit != 0 && len(m.items) >= m.sizeLimit {
+		return ErrSizeLimit
```

Comment:
If it is hard to handle from the caller of `Put` function, it is better to leave a log here instead of returning the error.

Reply:
Also, regarding the log, it would be better to leave a log with the information about the limit and `itemWithNonce`.

Reply:
In this PR, it is not hard to handle. There is no need to handle the error.
And If the log is needed, I think this level log is not proper, it is better for caller to make a log.
Because, this ItemSortedMap is for several purpose like bridge tx pool and handledEventMap.